flocker/provision/test/test_ssh_conch.py

```diff
@@ -71,7 +71,7 @@ def test_put(self):
 
         def check(_):
             self.assertEqual(self.server.home.child('file').getContent(),
-                             "hello")
+                             content)
```

Comment:
I'm leaving this for the moment because a similar test occurs elsewhere in the file, and consistency is good.